app/src/main/java/ar/com/wolox/android/training/ui/home/news/HomeNewsAdapter.kt

```diff
@@ -20,15 +34,23 @@ class HomeNewsAdapter @Inject constructor(private val newsList: List<HomeNews>,
     }
 
     override fun onBindViewHolder(holder: HomeNewsViewHolder, position: Int) {
-        var prettyTime = PrettyTime()
+        val prettyTime = PrettyTime()
 
         holder.newsTitle.text = newsList[position].title
+
         holder.newsText.text = newsList[position].text
-        holder.newsPicture.setImageURI(newsList[position].pictureString)
-        holder.newsDate.text = prettyTime.format(newsList[position].date)
-        holder.newsLike.setImageResource(R.drawable.ic_like_off)
-        holder.newsLike.tag = R.drawable.ic_like_off
 
+        holder.newsPicture.setImageURI(newsList[position].pictureString.replace("http://","https://"))
+
+        holder.newsDate.text = prettyTime.format(SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").parse(newsList[position].date))
```

Comment:
Add the SimpleDateFormatPattern to a constant and the use it here.
Maybe you should create the SimpleDateFormat Object before , and the use it here

Reply:
Solved in next PR update.